lib/http3/server.c

```diff
@@ -910,8 +914,9 @@ static int handle_input_expect_headers(struct st_h2o_http3_server_stream_t *stre
         return 0;
     }
 
-    { /* set priority */
-        assert(!h2o_linklist_is_linked(&stream->scheduler.link));
+    /* set priority */
+    assert(!h2o_linklist_is_linked(&stream->scheduler.link));
+    if (!stream->received_priority_update) {
```

Comment:
Just wanted to confirm my understanding here. If we have already received `PRIORITY_UPDATE` frame, it should override the priority information contained in the header, therefore we skip processing the priority header if that is the case. Am I right?

Reply:
Yes, a frame is considered an "update" of the priority information conveyed by the header field.